spec/integration/indexer_integration_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe 'indexer integration tests', :vcr do
+  it 'indexes donor tags when they exist' do
+    r = Spotlight::Resources::Purl.new(url: 'http://purl.stanford.edu/vw282gv1740') # Feigenbaum PURL with donor tags
+    solr_doc = r.to_solr.first
+    expect(solr_doc['donor_tags_tesim']).to eq ['Knowledge Systems Laboratory', 'medical applications', 'Publishing', 'Stanford', 'Stanford Computer Science Department']
+  end
+  it 'does not index donor tags when they do not exist' do
+    r = Spotlight::Resources::Purl.new(url: 'http://purl.stanford.edu/bd955gr0721') # Revs PURL without donor tags
```

Comment:
Line is too long. [114/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.